_projects/2023/100506380/100506380.Rmd

```diff
@@ -0,0 +1,480 @@
+---
+title: "The Phenomenon of Poverty"
+description: |
+  Visualizing the global distribution of disposable income between 1990 and 2019- Our World in Data.
+categories: "2023"
+author: "Frederick Peña Sims"
+date: "2023-12-19"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(out.width="100%", fig.align="center")
+```
+
+## The Phenomenon of Poverty
+
+### Introduction
+
+Poverty is a longstanding issue of known human societies ingrained into the logic of ['Capital and Ideology'](https://www.hup.harvard.edu/books/9780674980822). Be it out of colonial motivation, the United Nations, after the great catastrophe that was the second world war, set out its aim to diminish poverty globally and is now prominently presented as the first of 17 Sustainable Development Goals for 2030. While progress has been made, the Covid-19 pandemic, certainly, undid previously made progress. This project takes a look at the historical development of global distribution of the population between different poverty thresholds.
+
+### The original graph
+
+The original graph can be found on the web page of [Our World in Data](https://ourworldindata.org/grapher/distribution-of-population-between-different-poverty-thresholds-up-to-30-dollars?time=earliest..2008). It, visually, portrays how daily disposable income has been globally distributed between 1990 and 2019. One major strength of this graph is that, at a glance, one is able to see, tentatively, historical developments of each disposable income bracket in relation to each other.
+
+## Getting the data right
+
+### Packages
+
+```{r, echo=TRUE, results='hide'}
+library(tidyverse)
+library(ggplot2)
+library(gganimate)
+library(forcats)
+library(ggrepel)
+library(distill)
+```
+
+### Getting the data
+
+The data is easily accessible under the integrated download button below the graph on the previously indicated web page of [Our World in Data](https://ourworldindata.org/grapher/distribution-of-population-between-different-poverty-thresholds-up-to-30-dollars).
+
+```{r, echo=TRUE}
+data_raw <- read.csv("data/distribution-of-population-between-different-poverty-thresholds-up-to-30-dollars.csv")
```

Comment:
This data folder is not part of your PR. Please `git add` the whole contents of the `_projects` directory (including the `_files` folder that is generated), then commit and push.

EDIT: I see below that you may have a `test` folder. No need to add that one.

Reply:
I added the data folder and the relevant csv file. Could you tell me what you mean by the edit? I think I am not quite following.